src/pybind/mgr/dashboard/frontend/src/app/shared/components/critical-confirmation-modal/critical-confirmation-modal.component.ts

```diff
@@ -25,6 +25,8 @@ export class CriticalConfirmationModalComponent implements OnInit {
   itemDescription: 'entry';
   itemNames: string[];
   actionDescription = 'delete';
+  showInfoMessage = false;
+  infoMessage: string;
```

Comment:
one would be enough and I prefer `infoMessage`
```suggestion
  infoMessage: string;
```